protocol-definitions/custom/Custom.yaml

```diff
@@ -780,6 +780,34 @@ customTypes:
         type: MemberVersion
         nullable: false
         since: 2.0
+      - name: addressMap
+        type: Map_EndpointQualifier_Address
+        nullable: false
+        since: 2.1
+  - name: EndpointQualifier
+    returnWithFactory: true
```

Comment:
Yes. `EndpointQualifier` is not strictly an enum, but it has a pool of default instances, and we want to use those, instead of always creating a new instance during deserialization. So as far as I understand we need the factory.